src/components/time-picker/time-picker.st.css

```diff
@@ -0,0 +1,101 @@
+@namespace 'time-picker';
+
+:import {
+    -st-from: "../stepper/stepper.st.css";
+    -st-default: Stepper;
+}
+
+:import {
+    -st-from: "../../style/default-theme/variables.st.css";
+    -st-named:
+        BK2,
+        BK3,
+        PR2,
+        PR3;
+}
+:vars {
+    height: 36px;
+    width: 120px;
+}
+.root {
+    -st-states: focus, disabled;
+    font-family: arial;
+    font-size: 16px;
+    position: relative;
+    border: 1px solid value(BK3);
+    height: value(height);
+    width: value(width);
+    display: inline-flex;
+}
+
+.root:not(:disabled):hover {
+    border-color: value(PR3);
+}
+.root:focus {
+    border-color: value(PR2);
+}
+.root:disabled {
+    color: value(BK2);
+}
+
+.input {
+    font: inherit;
+    border: 0;
+    height: 24px;
```

Comment:
actually this is redundant rule. There is `height: 100%` a few line bellow. Will remove this